packages/@webex/plugin-meetings/src/config.js

```diff
@@ -91,7 +91,7 @@ export default {
       enableMediaNegotiatedEvent: false,
       enableUnifiedMeetings: false,
       enableAdhocMeetings: false,
-      enableTurnDiscovery: false,
+      enableTurnDiscovery: true, // Lets turn it on but provide a option to turn it off too
```

Comment:
should it be moved out of "experimental"?
maybe we should also change the default setting for the "Enable TURN discovery (experimental)" checkbox in the sample app to be enabled too? and remove the "(experimental)" bit from the string

Reply:
yep was planning to keep this for some time and remove the code completely 

will update the samples too